test/src/kbasesearchengine/test/search/ElasticIndexingStorageTest.java

```diff
@@ -1124,7 +1123,18 @@ public void addHighlighting() throws Exception {
         final Builder filter = MatchFilter.getBuilder();
         filter.withNullableFullTextInAll("multiWordInSearchMethod1 multiWordInSearchMethod2");
 
+
+        //highlight turned off would give null objects unless objectData is set to true
```

Comment:
Can you note that the purpose of this test is to show that turning highlighting off works?

Reply:
okay